### DISCUSSION TITLE
How to configure RaSCSI V1.1?

### DISCUSSION THREAD

**Discussion by DRBAUS** [akuker/RASCSI#1401] [General]
Hi there, 

A few years ago I purchased a Raspberry Pi Zero W V1.1 for use with my vintage samplers. 

Back then I was not able to configure because, from what I remember, it was not able to connect to my router to configure the unit. I left it at that because I still had some SCSI HDs that worked and did not bother. I am now starting on a new music project and could really use a smaller solution with regards to storage....and I still have the RaSCSI....so I want to give it another try.

Where do I start with this? How do I configure it so I can actually start using it.

Thanks in advance for you help.

Robin.

**Comment by rdmark** [comment]
Have you read through the resources on our wiki already?

If your device is already assembled and hooked up, a good starting point is https://github.com/PiSCSI/piscsi/wiki/Setup-Instructions

please let us know if you have any specific questions as you work through these instructions.
